### DISCUSSION TITLE
change password

### DISCUSSION THREAD

**Discussion by marcoka2004** [prasathmani/tinyfilemanager#516] [Q&A] [answered]
hello 
i have spent several hours trying to change the admin password and/or create new accounts (like paul and john)but i failed. 

These are the settings
-------

$auth_users = array(
    'admin' => '$2y$10$n3BI6.O1bK41j1bazW/ssuafjXpLwRjcLewH6FmUo2fBS.LxDCElK', //dogcatfish
    'user' => '$2y$10$Fg6Dz8oH9fPoZ2jJan5tZuv6Z4Kp7avtQ9bDfrdRntXtPeiMAZyGO', //12345
    'john' => '$2y$10$oM6BBYRmHm5e/zmz4VOT9OukN96VWfVkxBwccy5iaP2tkMs98vxja', //chocolate
'paul' => password_hash('aroundtheworld', PASSWORD_DEFAULT)
);

------
Hope somebody can help. (Great program btw)
regards

**Comment by EV-soft** [comment]
Are you writing settings to tinyfilemanager.php or config.php ?
If config.php exists the settings in this, will redefine settings in tinyfilemanager.php  !

**Comment by prasathmani** [comment, marked as answer]
The default configuration will be loaded from `config.php`, it is an additional configuration file, Feel free to remove completely this file and configure `"tinyfilemanager.php"`

**Comment by marcoka2004** [comment]
thanks for the help. It works now. I think it might be helpful if you write
this explaination in the documentation, it might help other users like me.

Thanks again

Marco

Il giorno mer 24 mar 2021 alle ore 22:24 Prasath Mani <
***@***.***> ha scritto:
